src/db/models/job.ts

```diff
@@ -38,13 +40,28 @@ export const PDF_FILE_RECORD = t.type({
   projectId: PROJECT_ID,
 });
 
+export const EXCEL_FILE_RECORD = t.type({
+  name: t.string,
+  planId: PLAN_ID,
```

Comment:
Why include `planId` here? I think you modeled this codec after `PDF_FILE_RECORD` (above), but we don't need the same information. Also, we cannot download Excel files in bulk, like we can do for PDF.

Since you only need file name, I suggest you drop this codec and add `fileName: t.string` to `JOB_METADATA_PROJECT_EXCE`